README.md

```diff
@@ -1,2 +1,17 @@
 # exchange-api-njs
-Nodejs-based api for exchanges
+
+## Introduction
+Nodejs-based api RChain RNode RPC.
+
+## Install
+
+> npm install --save git+https://git@github.com/rchain/exchange-api-njs.git
```

Comment:
To install from GitHub there is also a special syntax.
`npm install <githubname>/<githubrepo>[#<commit-ish>]`

Also peer dependencies must be installed first for generator to work.
```sh
npm install --save-dev grpc-tools protobufjs

npm install rchain/exchange-api-njs
```
The problem is also with the current version of `rnode-grpc-js` which fails to find npm _.bin_ folder unless it's a direct dependency.
I fixed the issue in `v0.8.4` version so please update it in the PR.